### DISCUSSION TITLE
AIO users - discussion

### DISCUSSION THREAD

**Discussion by PianSom** [springfall2008/batpred#646] [General]
Two questions for AIO users:
- what capacity have you set for predbat - 13.5kWh or 15.9kWh?
- has anyone worked out a charge curve for the AIO?

**Comment by springfall2008** [comment]
I think you need to use the battery scaling setting to tune down the reported capacity to 13.5Kwh 

For the charge curve you can probably find it quite easily by looking back at data for when you charged to 100% and work out the slow down for the last few percent.

**Comment by PianSom** [comment]
Thanks @springfall2008 . I have now changed Metric Battery Value Scaling (`input_number.predbat_metric_battery_value_scaling`) to 0.85 (= 13.58/15.96).  The input field does not look very happy - it is underlined in red.

For the charge curve, on data from last night I have calculated the following (on the basis of constant charge power, and time change between % changes is proportional to charge rate):
```
96 : 0.83
97 : 0.73
98 : 0.49
99 : 0.36
100 : 0.22
```
I'd be interested to see if anyone can confirm this sort of thing?

**Comment by PianSom** [comment]
Actually, if I look at the power draw in W compared to stable maximum then I get the following curve:
```
92 : 0.94
93 : 0.87
94 : 0.75
95 : 0.64
96 : 0.64 
97 : 0.41
98 : 0.29
99 : 0.18
100 : 0.18
```
This is perhaps a better curve?

**Comment by blastehh** [comment]
This is mine:
    93 : 0.9
    94 : 0.76
    95 : 0.65
    96 : 0.53
    97 : 0.41
    98 : 0.30
    99 : 0.18
    100 : 0.18

  battery_scaling: 0.85

**Comment by dannyoneill** [comment]
New predbat on AIO user here. Any other config tips for the AIO?

**Comment by gcoan** [comment]
> New predbat on AIO user here. Any other config tips for the AIO?

guidance for specific settings for the AIO in predbat is included in the inverter setup documentation https://springfall2008.github.io/batpred/inverter-setup/#givenergy-with-givtcp
